pootle/apps/pootle_translationproject/receivers.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.contrib.auth import get_user_model
+from django.core.mail import send_mail
+from django.db.models import Q
+from django.dispatch import receiver
+from django.template.loader import render_to_string
+from django.utils.translation import ugettext_lazy as _
+
+from .models import TranslationProject
+from .signals import tp_created_async, tp_creation_failed_async
+
+
+def get_recipients(project):
+    User = get_user_model()
+    return [user["email"] for user in User.objects.filter(
+        Q(permissionset__positive_permissions__codename="administrate",
+          permissionset__directory__pootle_path=project.pootle_path) |
+        Q(is_superuser=True)).values("email")]
```

Comment:
If you use `values_list("email", flat=True)` you can avoid the list comprehension.